pkg/sql/codegen/codegen_feature_column.go

```diff
@@ -41,34 +40,37 @@ func isXGBoostModule(module string) bool {
 	return strings.HasPrefix(module, "xgboost")
 }
 
-// MarshalToJSONOrDie converts any data to JSON string.
-// Exit the process if there is any error.
-func MarshalToJSONOrDie(in interface{}) string {
+// MarshalToJSONString converts any data to JSON string.
+func MarshalToJSONString(in interface{}) (string, error) {
```

Comment:
I don't think this function is necessary. We can use `json.Marshal`.

Reply:
This function is also used [here](https://github.com/sql-machine-learning/sqlflow/blob/develop/pkg/sql/codegen/tensorflow/codegen.go#L336), which would be used in [Python codegen template](https://github.com/sql-machine-learning/sqlflow/blob/develop/pkg/sql/codegen/tensorflow/template_train.go#L79).